verify.py

```diff
@@ -108,10 +118,20 @@ def __init__(self, configfile, auth, logger):
             elif key == "rdfLang":
                 self.lang = value
 
-        # if not specified in config, set default ext & lang to turtle
+        # if lang not specified in config, set the ext & lang to turtle
         if not hasattr(self, 'lang'):
```

Comment:
(Sorry for any confusion in my comments... I think I'm less confused now...)  `'lang'` should be `'rdfLang'`